oqs-template/generate.yml

```diff
@@ -366,17 +355,8 @@ kems:
     family: 'BIKE'
     variants:
       -
-        name: 'BIKE1_L1_CPA'
-        oqs_meth: 'OQS_KEM_alg_bike1_l1_cpa'
-      -
-        name: 'BIKE1_L3_CPA'
-        oqs_meth: 'OQS_KEM_alg_bike1_l3_cpa'
-      -
-        name: 'BIKE1_L1_FO'
-        oqs_meth: 'OQS_KEM_alg_bike1_l1_fo'
-      -
-        name: 'BIKE1_L3_FO'
-        oqs_meth: 'OQS_KEM_alg_bike1_l3_fo'
+        name: 'BIKE_L3'
```

Comment:
Another point of discussion from our dev meeting: is there a reason why BIKE_L1 was removed? It's enabled in v8, so it should work here as well(?)

Reply:
That's a mistake. I misread the equivalent change in the v8 branch and removed one too many. Will fix.